landing.css

```diff
@@ -0,0 +1,104 @@
+@import url("https: //fonts.googleapis.com/css2?family=Roboto");
+
+:root {
+  --main-primary-color: #1f59bb;
+  --main-secondary-color: rgba(53, 151, 245, 0.1);
+}
+
+/* Utils */
+
+.btn-primary {
+  background-color: var(--main-primary-color);
+}
+
+.btn-outline-primary {
+  border-color: var(--main-primary-color);
+}
+
+.background-primary {
+  background-color: var(--main-primary-color);
+}
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, "sans-serif";
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  color: #000000;
+}
+
+.wrapper {
+  background: #ffffff;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#ffffff" to be "#fff" (color-hex-length)](https://app.codacy.com/gh/codeclannigeria/CodeClanUI/pullRequest?prid=5545581)